test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+import testlib
+
+
+@testlib.skipImage("No wireguard-tools", "rhel-8-*", "centos-8-stream")
+class TestWireGuard(testlib.MachineCase):
+    provision = {
+        "machine1": {"address": "192.168.100.11/24", "restrict": False},
+        "machine2": {"address": "192.168.100.12/24", "restrict": False},
+    }
+
+    def testVPN(self):
+        m1 = self.machines["machine1"]
+        m2 = self.machines["machine2"]
+        b = self.browser
+
+        self.login_and_go("/network")
+
+
+        # Peer 2 (server)
+        if not m2.ostree_image:
+            m2.execute("firewall-cmd --add-port=51820/udp")
```

Comment:
That's the listening port of wireguard of that peer. I should probably use the variable instead of hard coding it.
```py
m2.execute(f"firewall-cmd --add-port={m2_listening_port}/udp")
```
Thanks for pointing it out.